version/5_6.sh

```diff
@@ -22,6 +22,7 @@ curl -sS http://getcomposer.org/installer | php
 chmod +x composer.phar
 mv composer.phar $HOME/.phpenv/versions/5.6/bin/composer
 
+sudo apt-get install php5.6-ldap
```

Comment:
which version finally gets installed? will apt-get not override all previous versions?

Reply:
previously the ldap is not available in php

Reply:
can we print the ldap version for each variation of php. the output is just showing a list of modules and im not sure if the correct version of ldap module is being installed for each php version

Reply:
![image](https://user-images.githubusercontent.com/31643910/44398213-b5c79700-a560-11e8-9587-54634a371927.png)